sales.html

```diff
@@ -1,15 +1,19 @@
 <!DOCTYPE html>
 
 <html>
-  <head>
+  <head>  
```

Comment:
Adding trailing whitespace is a sad diff! Make sure to look at the changes you're adding before you commit.